src/x86/cpuid.asm

```diff
@@ -0,0 +1,55 @@
+; Copyright © 2018, VideoLAN and dav1d authors
```

Comment:
It does not seem used from a first look and seems not needed.

Reply:
I will drop the first commit on the next push. This code is obviated by the Rust cpu-feature implementation.